neural_lam/train_model.py

```diff
@@ -130,41 +135,32 @@ def main(input_args=None):
 
     # Training options
     parser.add_argument(
-        "--ar_steps",
+        "--ar_steps_train",
         type=int,
-        default=1,
-        help="Number of steps to unroll prediction for in loss (1-19) "
-        "(default: 1)",
+        default=3,
+        help="Number of steps to unroll prediction for in loss function "
+        "(default: 3)",
```

Comment:
Yes, that should be 1 auto-regressive step. I think that is still the convention used:

https://github.com/mllam/neural-lam/blob/0a79c74cb1d6bf3e29b999188c0c6e4d9f95078b/neural_lam/weather_dataset.py#L194

So is this just changing the default to 3? I think it makes sense to keep this at 1, since that is typically the first step in training a weather model.

Reply:
Also, if we are doing changes to this naming: Would it not make more sense to call this `rollout_steps_train`? I feel that is typically how I refer to this when talking to people :smile:

Reply:
I am going to suggest we revert to the default value of `1` auto-regressive step (as in the model is only made to make a prediction for a single timestep) and we keep the name of the variable describing the number of auto-regressive steps as `ar_steps`. I can't remember why I increased the default to `3`, but I think we should stick with the current default to reduce the number complexity of this PR

Reply:
Changed back to default of 1 AR step in https://github.com/leifdenby/neural-lam/commit/b7bf506ae572ec4247d6ef5696c39803409d162e